quarkus/defaults/src/main/resources/application.properties

```diff
@@ -94,6 +94,8 @@ polaris.persistence.type=in-memory
 
 polaris.file-io.type=default
 
+polaris.events.type=default
```

Comment:
Does this mean only one listener is allowed? Are we plan to support multiple listeners? I believe we need to. One use case is that Polaris itself may ship a listener for audit purpose, while it still allows pluggable listeners from users.

Reply:
You could just write a listener that wraps both, like we do for the FileIOFactory if you want throttling and WASB translation.

Reply:
Each individual listener implementor can not wrap others, as it doesn't know if other listeners exist. We will need a way to let each to be added into a hub/wrapper.

Reply:
What's the use case where you don't know if others exist? In the example you gave, the third party would be aware of the listener that ships with Polaris

Reply:
People will know these OSS code for sure. To make it clear, it's an assumption that each listener implementor should not know other listeners exist. How Polaris organizes multiple listeners is a responsibility of Polaris server itself. Each individual listener should not be part of that business.

Reply:
What you wrote is true about most things we configure -- the persistence type, the realm resolver,  the rate limiter, the file IO factory, the authenticator, etc. In all of these cases, you plug in one implementation and that implementation is responsible for combining multiple semantics if you so desire.

Reply:
BTW, it's not a blocker. I'm OK if we want to support multiple listeners in followup PRs. But it's also worth to think ahead a bit how we are going to do that.

Reply:
I think we should ship a multi-listener wrapper in a followup PR, and I'd be happy to contribute that once we have alignment on the events here